### ISSUE TITLE
How to prepare data, mask, etc,...

### ISSUE THREAD

**Issue by MrTuanDao** [yisol/IDM-VTON#132]
Since you release training code, I really want to try finetune on my dataset but I cant understand how to prepare data

**Comment by luxiaolili** [comment]
> Since you release training code, I really want to try finetune on my dataset but I cant understand how to prepare data

you can prepare data follow  https://github.com/luxiaolili/IDM-VTON-train/
